README.md

```diff
@@ -1,4 +1,9 @@
 # VS Code Org Mode
+[![Version](https://vsmarketplacebadge.apphb.com/version/tootone.org-mode.svg)](https://marketplace.visualstudio.com/items?itemName=tootone.org-mode)
```

Comment:
duplicate

Reply:
@sipi, I have the same problem in my PRs. Your commit 8e80df9 looks like to be in each latest PR :)

Reply:
@sipi @rpeshkov I solved it by doing an interactive rebase and getting ride of the commit